.gitlab-ci.yml

```diff
@@ -157,7 +163,7 @@ builder-image:
   rules:
     - if: '$FAST_MODE == "true"'
       when: never
```

Comment:
should be here extra spaces?
such as:
```
- if: FAST MODE:
    when: never
when: on_success
```
?

Reply:
no, this part is correct I think